securethenews/blog/models.py

```diff
@@ -56,6 +56,6 @@ def posts(self):
         posts = BlogPost.objects.live().descendant_of(self)
 
         # Order by most recent date first
-        posts = posts.order_by('-date')
+        posts = posts.order_by('-id')
```

Comment:
@conorsch I hear you, but I feel pretty strongly about _not_ merging code that has known bugs. This is also such an easy case to encounter that I wouldn't describe it as an "edge" case.

This PR is still super worthwhile because the test is spot-on and should be kept as-is (well, plus an additional test to test the bug I described at the start of this thread). However, I think that the best fix is the first option I suggested in #97 (explicit is better than implicit!). That fix should be easy to implement and I'm happy to help @redshiftzero out with writing the migration (the only even slightly tricky part).